### ISSUE TITLE
Limit code coverage to "touched" classes in the `force test` subcomand.

### ISSUE THREAD

**Issue by joanbarros** [ForceCLI/force#259]
# Problem

The current behavior of the command is to display all the classes in the environment and all the test methods in the test class being ran. Something like the example below:

```
Coverage:

  0%  Class 1
  80%  Class 2
  100%  Class 3
  100%  Class 4
  0%  Class 5

Results:

  [PASS] Some_test_method
```

This is ok when you have a couple of dozens of classes but when you have A LOT of classes this begins being a problem. You start seeing that most of the classes listed are in 0% this is because you are running a test class that only affects an isolated part of your codebase.
## Solution

The proposed solution is to hide those classes in which the coverage is 0% and put a flag in place called something like `--all-coverage` if you'd like to see the current behavior.

**Comment by joanbarros** [comment]
Opinions are welcome on the name of the option to display all coverage or if it's needed at all!

**Comment by joanbarros** [comment]
Ohh and just an FYI, I'm currently working on a pull request for this feature. :)

**Comment by dcarroll** [comment]
Nice!

On Wed, Feb 17, 2016 at 12:45 PM, Joan Barros <notifications@github.com>
wrote:

> Ohh and just an FYI, I'm currently working on a pull request for this
> feature. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heroku/force/issues/259#issuecomment-185395980>.
>



-- 
Thanks,
Dave

**Comment by joanbarros** [comment]
Any feedback on wether the flag to display that all the other classes with zero coverage is necessary? I can't really seem to think of a scenario that you'd like to see all the classes that had 0% coverage.

**Comment by cwarden** [comment]
This was implemented in #267.
